bib/internal/podmanutils/podmanutils.go

```diff
@@ -0,0 +1,38 @@
+package podmanutils
```

Comment:
(super nitick) in the go standard library the convention seems to be to use the singular form (ioutil, httputil). So maybe just `podmanutil`?